src/components/ListView/index.js

```diff
@@ -0,0 +1,44 @@
+import { useWindowStore } from "../../store";
+import EventListPreview from "../EventListPreview";
+import FavoriteButton from "../FavoriteButton";
+import StyledLink from "../StyledLink";
+import StyledList from "../StyledList";
+import StyledListItem from "../StyledListItem";
+import StyledSection from "../StyledSection";
+import VenueListPreview from "../VenueListPreview";
+
+export default function ListView({ type, data }) {
```

Comment:
It used to be `EventListPreview` and `VenueListPreview` but I refactored both components and now it can do both. That's why I gave it such a generic name.